blueprints/vendor-shim/files/vendor/__name__-shim.js

```diff
@@ -0,0 +1,9 @@
+(function() {
```

Comment:
:ok_hand: @rwjblue.

Renamed and documented in https://github.com/ember-cli/ember-cli/pull/4935